website/docs/patterns/overview.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Patterns
+status: comingSoon
+---
+
+## What is a pattern?
+
+Pattern generally refers to a combination of components which work together to accomplish a function or render a layout and is often used to solve the same problem in a consistent manner. Common examples of patterns are forms, navigation, combinations of action buttons, etc.
+
+> "Components are an interface chunk to be added to an overall layout. whereas patterns may be UI or a variety of other things, like a behavior, flow, application motif or something else." ([Nathan Curtis](https://medium.com/eightshapes-llc/patterns-components-2ce778cbe4e8))
```

Comment:
This doesn't read very clearly, and your description provides enough essential detail. If the overall article feels useful, maybe link to it at the bottom of the page as an additional resource.

Reply:
I debated leaving that in there, happy to remove it though if my description is detailed enough!